eng/common/scripts/stress-testing/test.ps1

```diff
@@ -0,0 +1,16 @@
+
+function determineRetryTests ([ref] $val, [ref]$val2) {
```

Comment:
Did you mean to check this in? If so let's make it a pester test: https://github.com/Azure/azure-sdk-tools/blob/main/doc/development/powershell.md#unit-testing

Reply:
oh no, accident, it's my testing ps script